lib/ansible/modules/cloud/amazon/ec2_ami_copy.py

```diff
@@ -39,10 +39,10 @@
     description:
       - The name of the new AMI to copy.
     required: false
-    default: null
+    default: "default"
```

Comment:
as with requriements, changes in defaults should also be noted in the description with versions
- As of 2.3 the default is 'default', in prior versions it was 'null'.